ic-os/components/networking/nftables/hostos/setup-nftables.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# Substitute correct configuration parameters into nftables.conf.
+
+function usage() {
+    cat <<EOF
+Usage:
+  setup-nftables -n config.ini -i nftables.template -o nftables.conf
+
+  Generate nftables config from template file.
+
+  -n config.ini: network configuration description file
+  -i infile: input ic.json5.template file
+  -o outfile: output ic.json5 file
+EOF
+}
+
+# XXX: the following function is duplicate with generate-network-config.sh
+# -- consolidate
+#
+# Read the network config variables from file. The file must be of the form
+# "key=value" for each line with a specific set of keys permissible (see
+# code below).
+#
+# Arguments:
+# - $1: Name of the file to be read.
+function read_network_variables() {
+    # Read limited set of keys. Be extra-careful quoting values as it could
+    # otherwise lead to executing arbitrary shell code!
+    while IFS="=" read -r key value; do
+        case "$key" in
+            "ipv6_prefix") ipv6_prefix="${value}" ;;
+        esac
+    done <"$1"
+}
```

Comment:
All these "read______variables" functions are being removed after we roll out the new config 😈